content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -154,7 +154,8 @@ GDrive remote), for example:
 
 ```dvc
 $ dvc remote modify myremote gdrive_client_id 'client-id'
-$ dvc remote modify myremote gdrive_client_secret 'client-secret'
+$ dvc remote modify myremote --local \
```

Comment:
they are per project, not sensitive to that extent

Reply:
OK, rolled back this one.